### ISSUE TITLE
Cannot read property addListener on Electron

### ISSUE THREAD

**Issue by frankfabregat** [djipco/webmidi#101]
I was able to successfully add working listeners for a midi device on a browser (Microsoft Edge Chromium). However, when using Electron on Windows 10, I get the following error:
>Uncaught (in promise) TypeError: Cannot read property 'addListener' of undefined
    at WebMidi.<anonymous> (index.html:201)
    at WebMidi.onPortsOpen (C:\Users\user\Documents\GitHub\project\node_modules\webmidi\webmidi.min.js:31)
    at \<anonymous>

This is how I am using the api:
>var WebMidi = require('webmidi'); //This line is only used in Electron
 WebMidi.enable(function (err) {
        if (err) {
          console.log("WebMidi could not be enabled.", err);
        } else {
          console.log("WebMidi enabled!");
          var input = WebMidi.inputs[1];
            input.addListener('noteon', 1,
            function (e) {
              console.log("button Pressed");
            }
          );
}

I installed using NPM.

**Comment by djipco** [comment]
Your code tries to add a listener to the second input found (position 1). Is there such an input available on your machine? To be sure, you can use the code below to print out all available inputs:

```javascript
WebMidi.enable( function(error) {

  if (error) console.log(error);

  console.log(WebMidi.inputs);

});
```

**Comment by frankfabregat** [comment]
I do have an input available on my code. I changed `var input = WebMidi.inputs[1];` to `var input = WebMidi.getInputByName("MIDIIN2 (LPMiniMK3 MIDI)");` to make sure the number does not affect it. On Edge, this is what I get on the console:
![image](https://user-images.githubusercontent.com/19824295/72033008-9d8c1100-3256-11ea-8351-d3f819a508a8.png)
As you can see, after `WebMidi enabled!`, you can see `(2) [input, input]`. When I close Edge in order to prevent interference and open the Electron one, this is what I get:
![image](https://user-images.githubusercontent.com/19824295/72032847-438b4b80-3256-11ea-802a-de3c060105c1.png)
Instead of `(2) [input, input]`, I only get `[]`. However, if I open the array, I can see that it detects the MIDI device, but not in the way that I got in the browser:
![image](https://user-images.githubusercontent.com/19824295/72032891-5c93fc80-3256-11ea-985b-5c0eb227c3a7.png)
I tried setting my `input` variable to the name, input 2, and input 1, and all of those gave me the same error. 
In conclusion, it seems that it detects the input, but is not showing it the same for some reason.

**Comment by frankfabregat** [comment]
I am truly sorry. I am not very familiar with the console. Turns out I was getting the inputs before everything was being loaded. Thank you very much.

**Comment by djipco** [comment]
No problem. 

By the way, different browsers may report different names and IDs for the devices. I do not have control over that as these are coming straight from the OS MIDI layer.
